lib/logstash/inputs/tcp.rb

```diff
@@ -153,7 +153,7 @@ def handle_socket(socket, client_address, client_port, output_queue, codec)
       codec.decode(read(socket)) do |event|
         event["host"] ||= client_address
         event["port"] ||= client_port
-        event["sslsubject"] ||= socket.peer_cert.subject if @ssl_enable && @ssl_verify
+        event["sslsubject"] ||= socket.peer_cert.subject.to_s if @ssl_enable && @ssl_verify
```

Comment:
it was always an implicit rule to only have strings, numerics and `Timestamp` objects in the `Event` fields but this is somewhat made explicit in the Java `Event` because of the explicit Java<->Ruby type conversions. If a type is not know it will throw an exception.